remoteappmanager/cli/remoteappdb/__main__.py

```diff
@@ -89,7 +89,7 @@ def list(ctx):
                                          policy.volume_source,
                                          policy.volume_target,
                                          policy.volume_mode)
-                    for app, policy in orm.apps_for_user(session, user)]
```

Comment:
This comment is related to the functiion docstring. Please update it since it refers to users instead of images.